include/odp/api/spec/time.h

```diff
@@ -78,22 +108,33 @@ odp_time_t odp_time_local(void);
 odp_time_t odp_time_global(void);
 
 /**
- * Current local time in nanoseconds
+ * Current global time in nanoseconds
  *
- * Like odp_time_local(), but the time stamp value is converted into nanoseconds.
+ * Like odp_time_global(), but the time stamp value is converted into nanoseconds.
  *
- * @return Local time stamp in nanoseconds
+ * @return Global time stamp in nanoseconds
  */
-uint64_t odp_time_local_ns(void);
+uint64_t odp_time_global_ns(void);
 
 /**
- * Current global time in nanoseconds
+ * Current global time (strict)
  *
- * Like odp_time_global(), but the time stamp value is converted into nanoseconds.
+ * Like odp_time_global(), but reads the time stamp value more strictly (see
+ * odp_time_local_strict() documentation) in the program order.
+ *
+ * @return Global time stamp
+ */
+odp_time_t odp_time_global_strict(void);
+
+/**
+ * Current global time in nanoseconds (strict)
+ *
+ * Like odp_time_global_ns(), but reads the time stamp value more strictly (see
+ * odp_time_local_strict() documentation) in the program order.
```

Comment:
Or: `Like odp_time_global_strict(), but the time stamp value is converted into nanoseconds.`

Reply:
Fixed in v3.